From cd167517e51d15fc48935e94ce233fe24c782731 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@nxp.com>
Date: Fri, 28 Sep 2018 14:44:35 +0800
Subject: [PATCH 4870/5242] MLK-19850-3 ARM64: dts: fsl-imx8qm-device: enlarge
 USB PHY register region

commit  85828d6186d21d3ea29aa9b1009bba4c046de116 from
https://source.codeaurora.org/external/imx/linux-imx.git

There is a DCD module at USBPHY, the offset is from 0x800.

Reviewed-by: Jun Li <jun.li@nxp.com>
Signed-off-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-device.dtsi      |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
index 8cf06bc..0972f7b 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
@@ -3610,7 +3610,7 @@
 
 	usbphy1: usbphy@0x5b100000 {
 		compatible = "fsl,imx8qm-usbphy", "fsl,imx7ulp-usbphy", "fsl,imx6ul-usbphy", "fsl,imx23-usbphy";
-		reg = <0x0 0x5b100000 0x0 0x200>;
+		reg = <0x0 0x5b100000 0x0 0x1000>;
 		clocks = <&clk IMX8QM_USB2_PHY_IPG_CLK>;
 		power-domains = <&pd_conn_usbotg0_phy>;
 	};
-- 
1.7.9.5

